src/SelectField.tsx

```diff
@@ -5,17 +5,17 @@ import { SelectProps } from "antd/lib/select";
 import { FormikFieldProps } from "./FieldProps";
 
 export const SelectField = (
-  props: FormikFieldProps & SelectProps<any> & { children: React.ReactNode }
+  { name, validate, children, ...restProps }: FormikFieldProps & SelectProps<any> & { children: React.ReactNode }
 ) => {
   return (
-    <Field name={props.name} validate={props.validate}>
-      {({ field, form }: { field: any; form: FormikProps<any> }) => (
+    <Field name={name} validate={validate}>
+      {({ field: { name: fieldName, value: fieldValue }, form: { setFieldValue } }: { field: any; form: FormikProps<any> }) => (
```

Comment:
why the renaming of value?

Reply:
Renaming of `value` to `fieldValue` was to not confuse `onChange`'s arrow function's value parameter `value => setFieldValue( fieldName, value )` with the current field's value.

I'm fairly certain arrow function's parameters will be prioritized over outside scoped variables but this is purely for readability so we know which value is going where.

Reply:
Ok, these are kind of less important cosmetics, but I would suggest to leave the value provided by formik as is and in the change handler either use the same name (it will work as it shadows the variables of the outer scope) or to prevent name clashes rename the variable in the change handler (here just a **v** for value would be fine for me).

In any case if you leave as is I would also accept the merge.

Reply:
I will change as per your suggestion. That works totally fine for me.